### ISSUE TITLE
Release build as default

### ISSUE THREAD

**Issue by sourcebox** [burtonageo/cargo-bundle#103]
When just running `cargo bundle`, a debug build is used, so you have to run `cargo bundle --release` for a release build. IMO this should be the default, because bundles is normally used for distribution and debug builds are kind of rare in this situation.

**Comment by madsmtm** [comment]
I disagree, I think it should match `cargo build` as it does now, for consistency - otherwise I'll have to learn a `--debug` flag when I want to test my application as a bundle.

Though the documentation could spell out more explicitly that `--release` is needed.

**Comment by sourcebox** [comment]
On the other hand, I use cargo-wix to create the Windows installer and this will always use a release build without any flags.

**Comment by wiiznokes** [comment]
cargo install also use --release as default

**Comment by madsmtm** [comment]
I'll retract my earlier statement, I agree that this should default to a release build, like `cargo install`.
